frontend/src/components/chats/components/current-chat/components/messages-list/styles.module.scss

```diff
@@ -0,0 +1,11 @@
+.messagesListWrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 74.5vh;
```

Comment:
Can't we avoid vh here?

Reply:
We need fixed height to enable scroll, do not we?

Reply:
I was sure there is a way to achieve this without using vh, but if not let's leave it as it is